README.md

```diff
@@ -554,6 +555,7 @@ Table of Contents
 ## Email
 
   * [10minutemail](https://10minutemail.com) - Free, temporary email for testing.
+  * [Amail](https://www.amail.az) - Free 10 GB+ @amail.az email for free
```

Comment:
Thanks, generally we only put english sites on the list, do they perhaps have a english UI - I couldnt find any language links on there